app/sagas/user.ts

```diff
@@ -1,65 +1,50 @@
 import { call, put, takeLatest } from 'redux-saga/effects';
 
 import { userSlice } from '../redux/slices';
-import {
-  ILoginUserActionPayload,
-  IRegisterUserActionPayload,
-} from '../redux/types';
-import { apiInstance } from '../services/api';
-import {
-  IApiLoginUserPayload,
-  IApiRegisterUserPayload,
-} from '../services/api/apiPayload.types';
-import {
-  LoginUserResponse,
-  RegisterUserResponse,
-} from '../services/api/apiResponse.types';
-import { StaticNavigator } from '../services/navigator';
+import { IAuthUserActionPayload } from '../redux/types';
+import { loginUser, logout, registerUser } from '../services/firebase';
```

Comment:
let's add prefix for functions -- `firebaseLoginUser`, `firebaseLogout`, `firebaseRegisterUser`